spec/services/analytics_spec.rb

```diff
@@ -9,20 +9,23 @@
     }
   end
 
+  let(:ahoy) { instance_double(FakeAhoyTracker) }
+
+  before { allow(FakeAhoyTracker).to receive(:new).and_return(ahoy) }
```

Comment:
if we stubbed `Ahoy::Tracker` instead of `FakeAhoyTracker` here, would that make it possible to remove the env check in `Analytics` ?